src/applications/yellow-ribbon/components/FindYellowRibbonPage/index.js

```diff
@@ -44,8 +44,8 @@ export const FindYellowRibbonPage = () => (
         </ul>
 
         <p>
-          Search for schools participating in the current academic year by one
-          or all of the terms below.
+          Search for schools participating in the current academic year, August
+          2019 through July 2020.
```

Comment:
I decided not to use `moment` here as I feel like business requirements could change considerably before we'd ever need to update this again